static/app/views/issueList/utils.tsx

```diff
@@ -48,7 +48,9 @@ export function getTabs(organization: Organization) {
       },
     ],
     [
-      Query.FOR_REVIEW,
+      organization.features.includes('organizations:javascript-console-error-tag')
+        ? Query.FOR_REVIEW + ' empty_stacktrace.js_console:False'
+        : Query.FOR_REVIEW,
```

Comment:
I think we'd want to add something to the Query enum or change it somehow, we have a lot of places that check if we're on the inbox tab by checking `query === Query.FOR_REVIEW` I think this enables/disables the inbox specific sort and a few other inbox features